langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/statements/foreach/ForeachNegativeTests.java

```diff
@@ -30,10 +30,10 @@
  */
 public class ForeachNegativeTests {
 
-    @Test(enabled = false)
+    @Test
```

Comment:
I feel it's better to leave this disabled, since it has unsupported features (functional binding pattern).

Otherwise we will lose the track of this, since this is enabled..

Reply:
we can make a note of it in the excel sheet and create an issue?

Reply:
docs tends to get lost in time :)
Lets keep it disabled for now.. Since we have done the same thing for error related langlib tests, I think its good to follow a single pattern..
We can enable it once we add the func-binding-pattern

Reply:
alright :)